Default (Linux).sublime-keymap

```diff
@@ -35,14 +35,16 @@
   { "keys": ["ctrl+k", "ctrl+alt+down"], "command": "create_pane_with_file", "args": {"direction": "down"} },
   { "keys": ["ctrl+k", "ctrl+alt+left"], "command": "create_pane_with_file", "args": {"direction": "left"} },
 
-
   // You can pull a file from another pane by binding the following command:
   // { "keys": [], "command": "pull_file_from_pane", "args": { "direction": ""} }
 
-  { "keys": ["ctrl+k", "ctrl+z"], "command": "zoom_pane", "args": {"fraction": 0.9} },
-  { "keys": ["ctrl+k", "ctrl+shift+z"], "command": "unzoom_pane", "args": {} },
+  // NOTE: The following keybindings are disabled as they conflict with built-in features.
```

Comment:
technically, the built-in ones conflict with these ones because these came first ;)

Reply:
A historical detail and not really relevant to the user seeing this message. :)